<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AconityClass</class>
 <widget class="QMainWindow" name="AconityClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1066</width>
    <height>783</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aconity</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1051</width>
      <height>771</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tbMain">
     <attribute name="title">
      <string>Main</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1031</width>
        <height>731</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_19">
       <item>
        <widget class="QLabel" name="mainlblimage">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_20">
           <item>
            <widget class="QComboBox" name="maincbBitwise"/>
           </item>
           <item>
            <widget class="QPushButton" name="mainpbBitwise">
             <property name="text">
              <string>Bitwisee</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_29">
           <item>
            <widget class="QComboBox" name="maincbStructType"/>
           </item>
           <item>
            <widget class="QLabel" name="label_29">
             <property name="text">
              <string>Art des Strukturelements</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_30">
           <item>
            <widget class="QSpinBox" name="mainsbStructSizeX">
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_30">
             <property name="text">
              <string>Größe X</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="mainsbStructSizeY">
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_31">
             <property name="text">
              <string>Größe Y</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_31">
           <item>
            <widget class="QComboBox" name="maincbOperation"/>
           </item>
           <item>
            <widget class="QLabel" name="label_32">
             <property name="text">
              <string>Operation</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="mainpbShowFirst">
             <property name="text">
              <string>Show</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_32">
           <item>
            <widget class="QComboBox" name="maincbStructType_2"/>
           </item>
           <item>
            <widget class="QLabel" name="label_33">
             <property name="text">
              <string>Art des Strukturelements</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_33">
           <item>
            <widget class="QSpinBox" name="mainsbStructSizeX_2">
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_34">
             <property name="text">
              <string>Größe X</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="mainsbStructSizeY_2">
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_35">
             <property name="text">
              <string>Größe Y</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_34">
           <item>
            <widget class="QComboBox" name="maincbOperation_2"/>
           </item>
           <item>
            <widget class="QLabel" name="label_36">
             <property name="text">
              <string>Operation</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="mainpbShowSecond">
             <property name="text">
              <string>Show</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="mainpbMask">
           <property name="text">
            <string>mask</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mainpbContour">
           <property name="text">
            <string>Kontur</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tbImgOne">
     <attribute name="title">
      <string>Bild 1</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1031</width>
        <height>731</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="tb1lblimage">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="tb1leLoad"/>
           </item>
           <item>
            <widget class="QPushButton" name="tb1pbLoadImage">
             <property name="text">
              <string>Load Image</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QSpinBox" name="tb1sbThreshold">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Threshold</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="tb1cbOTSU">
             <property name="text">
              <string>Otsu</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="tb1cbInverted">
             <property name="text">
              <string>inverted</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="tb1pbShowThresh">
             <property name="text">
              <string>show threshold</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QComboBox" name="tb1cbStructType"/>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Art des Strukturelements</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QSpinBox" name="tb1sbStructSizeX">
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Größe X</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="tb1sbStructSizeY">
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Größe Y</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QComboBox" name="tb1cbOperation"/>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Operation</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="tb1pbShowFirst">
             <property name="text">
              <string>Show</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QComboBox" name="tb1cbStructType_2"/>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Art des Strukturelements</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QSpinBox" name="tb1sbStructSizeX_2">
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Größe X</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="tb1sbStructSizeY_2">
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Größe Y</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QComboBox" name="tb1cbOperation_2"/>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Operation</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="tb1pbShowSecond">
             <property name="text">
              <string>Show</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tbImgTwo">
     <attribute name="title">
      <string>Bild 2</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1031</width>
        <height>731</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="tb2lblimage">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLineEdit" name="tb2leLoad"/>
           </item>
           <item>
            <widget class="QPushButton" name="tb2pbLoadImage">
             <property name="text">
              <string>Load Image</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QSpinBox" name="tb2sbThreshold">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Threshold</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="tb2cbOTSU">
             <property name="text">
              <string>Otsu</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="tb2cbInverted">
             <property name="text">
              <string>inverted</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="tb2pbShowThresh">
             <property name="text">
              <string>show threshold</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QComboBox" name="tb2cbStructType"/>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Art des Strukturelements</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <item>
            <widget class="QSpinBox" name="tb2sbStructSizeX">
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Größe X</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="tb2sbStructSizeY">
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Größe Y</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_15">
           <item>
            <widget class="QComboBox" name="tb2cbOperation"/>
           </item>
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Operation</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="tb2pbShowFirst">
             <property name="text">
              <string>Show</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_16">
           <item>
            <widget class="QComboBox" name="tb2cbStructType_2"/>
           </item>
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Art des Strukturelements</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_17">
           <item>
            <widget class="QSpinBox" name="tb2sbStructSizeX_2">
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Größe X</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="tb2sbStructSizeY_2">
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>Größe Y</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_18">
           <item>
            <widget class="QComboBox" name="tb2cbOperation_2"/>
           </item>
           <item>
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Operation</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="tb2pbShowSecond">
             <property name="text">
              <string>Show</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Aconity.qrc"/>
 </resources>
 <connections/>
</ui>
